assets/js/common/Table/Table.jsx

```diff
@@ -82,6 +83,8 @@ function Table({
     (c) => c.filter && c.filterFromParams
   );
 
+  const hasFilters = columns.filter(({ filter }) => Boolean(filter)).length > 0;
```

Comment:
Nice oneliner bro

Reply:
it is the freaking npm formatter, it doesn't like anything that it isn't this single line :see_no_evil: